go/common/sgx/pcs/quote.go

```diff
@@ -31,8 +31,21 @@ const (
 
 	// ppidDataLen is the PPID certification data length in bytes.
 	ppidDataLen = 404
+
+	// DefaultMinTCBEvaluationDataNumber is the default minimum TCB evaluation data number.
+	DefaultMinTCBEvaluationDataNumber = 12 // As of 2022-08-01.
```

Comment:
what's this

Reply:
What do you mean? It is the current TCB evaluation data number as of 2022-08-01. No reason to allow lower versions as in the future these can only increment with each TCB recovery.

> A monotonically increasing sequence number changed when Intel updates the content of the TCB evaluation data set: TCB Info, QE Idenity and QVE Identity. The tcbEvaluationDataNumber update is synchronized across TCB Info for all flavors of SGX CPUs (Family-Model-Stepping-Platform-CustomSKU) and QE/QVE Identity. This sequence number allows users to easily determine when a particular TCB Info/QE Idenity/QVE Identiy superseedes another TCB Info/QE Identity/QVE Identity (value: current TCB Recovery event number stored in the database).

Reply:
oh thx for the description